TriblerGUI/widgets/trustpage/style.css

```diff
@@ -7,11 +7,26 @@ html, body {
     background-color: #202020;
 }
 
+.metric {
```

Comment:
no not plural since the usage corresponds to a single metric. when this class would apply to the table itself then .metrics makes sense